dev-client/src/translations/en.json

```diff
@@ -566,9 +566,9 @@
     "required_method": "{{method}} (required)",
     "collection_method": {
       "slope": "Slope and Landscape",
-      "soilTexture": "Soil Texture",
+      "soilTexture": "Texture Class",
       "rockFragmentVolume": "Rock Fragment",
-      "soilColor": "Soil Color",
+      "soilColor": "Color",
```

Comment:
This change was not specified in the ticket, can you make a note somewhere explaining that this property label was also out of sync?

Reply:
hmm. that was in the Figma design. Going to re-review all of this with Derek and Courtney.